app_dart/lib/src/foundation/utils.dart

```diff
@@ -88,7 +89,28 @@ Future<RepositorySlug> repoNameForBuilder(List<Map<String, dynamic>> builders, S
 Future<List<Map<String, dynamic>>> getBuilders(HttpClientProvider branchHttpClientProvider, Logging log,
     GitHubBackoffCalculator gitHubBackoffCalculator, String bucket) async {
   final String filename = bucket == 'try' ? 'luci_try_builders.json' : 'luci_prod_builders.json';
-  String builderContent = await remoteFileContent(branchHttpClientProvider, log, gitHubBackoffCalculator, filename);
+  String builderContent = await remoteFileContent(
+      branchHttpClientProvider, log, gitHubBackoffCalculator, '/flutter/cocoon/master/app_dart/dev/$filename');
```

Comment:
Do we need to replace cocoon with the repo name?

Reply:
This is the old strategy with the fixed cocoon. Once changes merged, we will remove this function.

Reply:
Please add a todo here, mentioning that will be removed.

Reply:
Added.